ts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nationalarchives/tdr-generated-graphql",
-  "version": "1.0.11",
+  "version": "1.0.12",
```

Comment:
Do we have to manually update this, or does it get bumped automatically like the Scala package version?

Reply:
Correct. Have removed this change.